crates/frontend/src/pages/context_override.rs

```diff
@@ -335,20 +335,30 @@ pub fn context_override() -> impl IntoView {
                                 (context.clone(), overrides)
                             })
                             .collect::<Vec<(Context, Map<String, Value>)>>();
-                        ctx_n_overrides
-                            .into_iter()
-                            .map(|(context, overrides)| {
-                                view! {
-                                    <ContextCard
-                                        context=context
-                                        overrides=overrides
-                                        handle_edit=handle_context_edit
-                                        handle_clone=handle_context_clone
-                                        handle_delete=handle_context_delete
-                                    />
-                                }
-                            })
-                            .collect_view()
+                        if ctx_n_overrides.is_empty() {
+                            view! {
+                                <div class="flex font-semibold items-center justify-center">
```

Comment:
@pratikmishra356 Can we check if this is required in any other screen?

Reply:
Also can this be vertically centered?

Reply:
I checked this is not required in other screen, 
I think it should not be vertically centre as one had to look down a bit, 
since overrides text and create button is on top it is better to be aligned towards top

Reply:
You can make the text bigger, and then center it. May need an icon as well

Reply:
sure checking